esphome/components/aqi/abstract_aqi_calculator.h

```diff
@@ -3,12 +3,12 @@
 #include <cstdint>
 
 namespace esphome {
-namespace hm3301 {
+namespace aqi {
```

Comment:
```suggestion
namespace esphome::aqi {
```
Should update all namespaces and the comments